src/lib/controllers/download-controller.js

```diff
@@ -1,25 +1,33 @@
 import DownloadFileOneColumn from '@/lib/download/download-file-one-column.js'
 
 export default class DownloadController {
-  static download (downloadType, data) {
+  static download (downloadType, data, l10n) {
     switch (downloadType) {
       case 'plainSourceDownload':
-        return this.plainSourceDownload(data)
+        return this.plainSourceDownload(data, l10n)
       default:
-        console.error(`Download type ${downloadType} is not defined.`)
+        if (l10n) {

```

Comment:
why do we need this test? If there is no language bundle shouldn't we just fail? It seems like a lot of unnecessary work to duplicate all the messages

Reply:
I am not sure but it is for errors to be sure that they would be printed.
But if you think that it is unneccesary - I would remove.

Reply:
let's remove it. I think if we don't have the l10n library the code should fail.

Reply:
done